bread/views/error.py

```diff
@@ -0,0 +1,78 @@
+import sys
+
+import htmlgenerator as hg
+from django.http import HttpResponse, HttpResponseNotFound
+from django.utils.translation import gettext_lazy as _
+
+from bread.layout import error as layouts
+
+
+def view400(request, exception):
+    return HttpResponse(
```

Comment:
When you call a function that has a ```@aslayout``` decorator, the response should already be an HttpResponse, so no need to wrap it here. Same should be true for the other views.
In order to set the status code for the response I think you should be able to do something like this:

```python
resp = layout.error_layout(.....)
resp.status = 400
return resp
```

Reply:
I didn't know about that. Thank you.